charts/zeebe-operate-helm/templates/deployment.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "zeebe-operate.fullname" . }}
   labels:
     app: {{ include "zeebe-operate.fullname" . }}
+    {{- toYaml  .Values.labels | nindent 4 }}
```

Comment:
TIL about nindent :) but I think newline is not really necessary or?

Reply:
Newline fixed. I added a validation for if there is no custom labels.

Reply:
Not 100% what I meant but is fine i think :)